changelog/january2025/2025-01-08-iam-deprecated-deprecation-of-the-resellercustomerbilli.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Deprecation of the ResellerCustomerBillingManager permission set
+status: deprecated
+author:
+    fullname: 'Join the #iam channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2025-01-08
+category: security-identity
+product: iam
+---
+
+Since January 8th 2025, the permission set ResellerCustomerBillingManager cannot be used in policies. This permission set was only available for clients that consume Scaleway Services through a reseller.
```

Comment:
```suggestion
Since January 8th, 2025, the permission set `ResellerCustomerBillingManager` cannot be used in policies. This permission set was only available for clients that consume Scaleway services through a reseller.
```